ui-admin/src/study/kits/KitList.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
+import {
+  ColumnDef,
+  getCoreRowModel, getFilteredRowModel, getSortedRowModel, useReactTable
+} from '@tanstack/react-table'
+
+import Api, { KitRequest } from 'api/api'
+import { StudyEnvContextT } from 'study/StudyEnvironmentRouter'
+import LoadingSpinner from 'util/LoadingSpinner'
+import { basicTableLayout } from 'util/tableUtils'
+import { instantToDateString, isoToInstant } from 'util/timeUtils'
+
+const pepperStatusToHumanStatus = (dsmStatus?: string): string => {
+  const statusMap: Record<string, string> = {
+    'CREATED': 'Created',
+    'LABELED': 'Prepared',
+    'SCANNED': 'Sent',
+    'RECEIVED': 'Returned'
```

Comment:
On the left are my guesses as to what Pepper might send for `currentStatus` (unless we decide to do away with that field since it's just derived from the individual activity dates). On the right are my guesses as to what this really means from the point of view of a study admin. For example, `SCANNED` means that the kit has been scanned on the way out... effectively having been shipped to the participant while `RECEIVED` means that GP has gotten the kit back, i.e. "returned" from the participant (as opposed to "received" by the participant).

Reply:
Also, I realize this is not very DRY. This UI is definitely going to be refined in the near future which could affect the de-dupe efforts, so I'm leaving this as-is for now.